rust-src/concordium_base/src/lib.rs

```diff
@@ -32,6 +32,8 @@ pub mod ps_sig;
 
 pub mod dodis_yampolskiy_prf;
 
+pub use ed25519_dalek;
```

Comment:
I don't think we want to re-export the entire crate. That's too much.

We should only export the public and secret keys and signatures since they appear in our own API.